l10n_ec_ote/README.rst

```diff
@@ -0,0 +1,114 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================
+Ecuador's Geopolitical Information
+===================================
+
+This module adds the information requiered to select partner's location in states, canton and parish.
+
+Infomación Geopolitica de Ecuador
+=================================
+
+Este módulo agrega la información necesaria para agregar la información de provincias, cantones
+y parroquias en la ubicación de clientes y proveedores.
+
+Installation
+============
+
+To install this module, you need to:
+
+* Add the module to your addons path.
+* Install the module as usual.
+
+Instalación
+===========
+
+Para instalar este módulo se debe:
+
+* Agregar el módulo al directorio de addons.
+* Instalarlo de manera regular.
+
+Configuration
+=============
+
+This module doesn't requiere any configuration.
+
+Configuración
+=============
+
+Este módulo no requiere configuraciones.
+
+Usage
+=====
+
+This module changes the structure in partner views to give you the options to select the cantons and parish, just use them.
+
+Instrucciones de uso
+====================
+
+Este módulo modifica la estructura de la vista de terceros, permitiendo seleccionar las provincias, cantones y parroquias, solo debes usarlas.
+
+Demostración en runbot
+======================
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/repo/github-com-oca-l10n-ecuador-212
```

Comment:
Just put https://runbot.odoo-community.org/runbot/212/9.0. People will directly get prompted with the login screen.